src/AdminConsole/EventLog/DTOs/OrganizationEventDto.cs

```diff
@@ -9,4 +10,27 @@ public record OrganizationEventDto(
     Severity Severity,
     string Subject,
     int OrganizationId,
-    DateTime PerformedAt);
\ No newline at end of file
+    DateTime PerformedAt)
+{
+    public OrganizationEvent ToNewEvent() =>
+        new()
+        {
+            Id = Guid.NewGuid(),
+            PerformedAt = PerformedAt,
+            EventType = EventType,
+            Message = Message,
+            Severity = Severity,
+            PerformedBy = PerformedBy,
+            Subject = Subject,
+            OrganizationId = OrganizationId
+        };
+
+    public EventLogEvent ToResponse() =>
+        new(PerformedAt,
+            EventType.ToString(),
+            Message,
+            Severity.ToString(),
+            PerformedBy,
+            Subject,
+            string.Empty);
```

Comment:
Transformed these from extension methods to member methods because it didn't make much sense to separate them